Feature: Setup project readme with requirements and specifications

jdmedlock opened the pull request:
Setup project readme with requirements and specifications

Resolves: N/a
See also: N/a

jdmedlock committed:
Feature: Setup project readme with requirements and specifications

Setup project readme with requirements and specifications

Resolves: N/a
See also: N/a

johnmeade-webdev reviewed (changes requested):
Line 39 currently reads - 'obtains results by through an api'; we should omit the word 'by'
Line 50 - we may want to just comment that github does off a safe way to input your API keys, just not through pushing it to the repo

jdmedlock committed:
Feature: Updated to address comments from PR review

Updated to address comments from PR review:
Line 39 currently reads - 'obtains results by through an api'; we should omit the word 'by'
Line 50 - we may want to just comment that github does off a safe way to input your API keys, just not through pushing it to the repo

Resolves: #1
See also: N/a
